bedevere/bpo.py

```diff
@@ -154,3 +172,12 @@ def create_hyperlink_in_comment_body(body):
             leftover_body = leftover_body[presence:]
     new_body = new_body + leftover_body
     return new_body
+
+
+async def _validate_issue_number(issue_number):
+    """Make sure the issue number exists on bugs.python.org."""
+    url = f"https://bugs.python.org/issue{issue_number}"
+    async with aiohttp.ClientSession() as session:
+        async with session.head(url) as res:
+            return res.status != 404
```

Comment:
Isn't it more true that as long as it isn't a 40x status? Or is there no way to get anything but a 404 back?

Reply:
There are other 40x status codes that correspond to a client error, that does not have to with the page not being found, i.e. `408 Request Timeout`. If bpo follows these conventions, then 404 should be the only possible status code to receive in the response.